src/handlers/hello.rs

```diff
@@ -0,0 +1,5 @@
+use actix_web::Responder;
+
+pub async fn hello() -> impl Responder {
+    String::from("Hello!")
+}
```

Comment:
This is temporal, and must go away when we begin to define handlers/endpoints, its just an example and it also helps us having a valid module to import